docs: add archive environment feature related posts (#686)
devopness/devopness: docs/environments/find-archived-environment.md

---
draft: true

> title: Find an Archived an Environment

Reviewer: ```suggestion
title: Find an Archived Environment
```

Author: Done

> intro: Sometimes you may need to inspect which environments have been archived recently. Access the archived environments list to check on all archived environments for the current project.

Reviewer: 1. Same notes on previous comments: the scenario been set is too specific, should be more broader, IMHO, so the readers can identify themselves with the topic even when their use case is not exactly like the one(s) exemplified in the documentation
2. The steps here are stopping after the user access the list. I don't believe we follow these pattern in any other topics written so far. I'd suggest guiding the reader to see the details of an environment. If halfway users are happy enough to see just a list, it's up to them to stop on that step, but that should not be the goal of this topic

```suggestion
intro: Sometimes you may need to access the details of an archived environment or see a list of environments in a project that have been archived. You can access details of all archived environments in a project owned by you.
```

Author: @jfoliveira the reason behind the post "cutting short" was because, in the proposed UX design, there's no indication of the user being able to access an archived environment without unarchiving it; I've tried to document the flow that was already described, without adding steps.

In the archived environment list, the only action available on the archived environment card is "unarchive".

Although, as we are documenting this use case early, we can propose changes using this documentation as a means to validate it.

As such, I believe the unarchive flow would be a better fit in a similar flow as to "archive": in the environment details, click `...` and click `Unarchive`

I'll update the PR with this new mindset

Author: Updated posts

---
draft: true
title: Find an Archived Environment
intro: Sometimes you may need to access the details of an archived environment or see a list of environments in a project that have been archived. You can access details of all archived environments in a project owned by you.
links:
    overview:
    quickstart:
    previous:
    next:
    guides:
    related:
    featured:
---

1. On Devopness upper-left corner, click the Devopness logo to see a list of existing projects
1. Select a `Project`
1. Find the `Environments` card
1. Click `View` in the `Environments` card to see a list of existing `Environments`

> 1. On the upper-right corner of the `Environment` resources list, click `ARCHIVED ENVIRONMENTS`

Reviewer: **TO DO**: update these instructions once `archive` feature is released.

1. Find the `Archived Environment` you want to inspect and click `View`

> 1. The `Environment` resources list will be displayed containing all the environment's running resources

Reviewer: `running resources`
We don't have this concept in Devopness.
Do you mean in a server is stopped (via Devopness or outside Devopness), therefore no longer `running`, that server will not be displayed in this list? 🤔

Author: You are correct, that was a misunderstanding of the intent design from my behalf; will apply the patch below, if there are any regards after the changes, I'm happy to continue the discussion in this thread or a new one.

```suggestion
1. The `Environment` resources list will be displayed containing all the environment's resources
```
